diff --git a/core/modules/rest/src/EventSubscriber/ResourceResponseSubscriber.php b/core/modules/rest/src/EventSubscriber/ResourceResponseSubscriber.php
index 9b5d24836dbb..5cc7dc970a7d 100644
--- a/core/modules/rest/src/EventSubscriber/ResourceResponseSubscriber.php
+++ b/core/modules/rest/src/EventSubscriber/ResourceResponseSubscriber.php
@@ -102,6 +102,13 @@ public function getResponseFormat(RouteMatchInterface $route_match, Request $req
     $requested_format = $request->getRequestFormat();
     $content_type_format = $request->getContentType();

+    // Workaround for if there is no formats for a DELETE call:
+    // make the response formats match the format of the request
+    if ($request->getMethod() === 'DELETE') {
+      if (empty($acceptable_request_formats)) $acceptable_request_formats = [$requested_format];
+      if (empty($acceptable_formats)) $acceptable_formats = [$requested_format];
+    }
+
     // If an acceptable response format is requested, then use that. Otherwise,
     // including and particularly when the client forgot to specify a response
     // format, then use heuristics to select the format that is most likely
